sandbox/libs/authn/src/main/java/org/opensearch/authn/User.java

```diff
@@ -61,4 +61,19 @@ public void setAttributes(Map<String, String> attributes) {
     public String toString() {
         return "User [username=" + username + ", bcryptHash=" + bcryptHash + ", attributes=" + attributes + "]";
     }
+
+    @Override
+    public String getHash() {
```

Comment:
Why not rename the other methods `getBcryptHash` and `setBcryptHash` to `getHash` and `setHash` respectively?